classes/submission/SubmissionDAO.inc.php

```diff
@@ -517,7 +517,7 @@ function deleteByContextId($contextId) {
 
 	/**
 	 * Delete the attached licenses of all submissions in a context.
-	 * @param $submissionId int
+	 * @param $contextId int
 	 */
 	function deletePermissions($contextId) {
```

Comment:
Looks like this method is no longer used anywhere. Can you remove it?

Reply:
ok, was unsure if it would be still needed. Removed now.